acceptance_tests/Dockerfile

```diff
@@ -1,8 +1,6 @@
 FROM camptocamp/c2cwsgiutils:5
 
-COPY . /acceptance_tests/
-
 WORKDIR /acceptance_tests
+COPY . ./
 
-CMD tail -f /dev/null
-CMD py.test -vv --color=yes --junitxml /tmp/junitxml/results.xml
+CMD ["tail", "-f", "/dev/null"]
```

Comment:
Why no more py.test ? It's done elsewhere ?

Reply:
In fact, this file was not used I just do some cleaning then this file is used again :-)